ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -106,7 +106,7 @@ CREATE TABLE clusterconfig (
 CREATE TABLE ambari_configuration (
   category_name VARCHAR(100) NOT NULL,
   property_name VARCHAR(100) NOT NULL,
-  property_value VARCHAR(2048),
+  property_value VARCHAR(4000) NOT NULL,
```

Comment:
If we are going with the `TEXT` type for Postgres, would it be better to go the `TEXT` route for all DB flavors to keep things consistent?

Reply:
Makes sense. This would mean:
- CLOB for Oracle
- LONGTEXT for MySQL/MariaDB

I'm not sure if we need to do this for unsupported DBs at all...

Reply:
However, given that we need to update the upgrade class too I'd change it like set the length in Postgres to 4k too.

Any objection?

Reply:
Nope.. no objection.

Reply:
Done